examples/vision/depth_estimation.py

```diff
@@ -25,17 +25,22 @@
 """
 
 import os
+
+os.environ["KERAS_BACKEND"] = "tensorflow"
```

Comment:
If it works with all backends, then you can remove this statement

Reply:
It still uses few TF images. `tf_image.convert_image_dtype`, `tf_image.image_gradients`,`tf_image.ssim`.

Can I just `import tensorflow `and remove the line `os.environ["KERAS_BACKEND"] = "tensorflow"` ?

Reply:
But if you switch to a different backend, does the code work? It's fine if there's some usage of TF as long as it's only in the data preprocessing section.

Reply:
It seems won't workout with other backend since `tf_image.image_gradients` and `tf_image.ssim` being used in `compute_loss` function(through calculate_loss).

I just tested for confirmation with pytorch backend. It is getting error at below line:

`dy_true, dx_true = tf_image.image_gradients(target)
`

> AttributeError: 'numpy.ndarray' object has no attribute 'get_shape'


Then I tried to convert it into tensor using `keras.ops.convert_to_tensor(target)` but got error below.

> AttributeError: 'Tensor' object has no attribute 'get_shape'

Then I tried to convert it into tensor using `tf.convert_to_tensor(target)` but got again error below.

> TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.